ui/lib/pki/addon/templates/issuers/index.hbs

```diff
@@ -93,7 +93,7 @@
     </LinkedBlock>
   {{/each}}
 {{else}}
-  <EmptyState @title="PKI not configured" @message={{this.message}}>
+  <EmptyState @title="PKI not configured" @message={{this.notConfiguredMessage}}>
```

Comment:
Thanks for updating the name here! Should we update the name of this controller property in the other routes?

Reply:
Good idea! I'll go back and make all of them more specific :)